src/test/resources/application.conf

```diff
@@ -1,4 +1,6 @@
 etc-client {
+  keystore-dir = "/tmp/etc-client/keystore"
```

Comment:
Minor detail: Should we use a different `keystore-dir` for our tests? If not we currently clear the `KeyStore` of our node while running the `KeyStoreImpl` tests.